Makefile

```diff
@@ -0,0 +1,26 @@
+MOCHA_REPORTER = spec
+UNIT_TESTS = $(shell find test/app -name "*.test.js")
```

Comment:
A Makefile - impressive! I hear this is the future - rather than grunt/gulp/etc.